block/produce.go

```diff
@@ -76,16 +76,16 @@ func (m *Manager) ProduceBlockLoop(ctx context.Context, bytesProducedC chan int)
 			}
 
 			bytesProducedN := block.SizeBytes() + commit.SizeBytes()
-			m.logger.Info("New block.", "size", uint64(block.ToProto().Size()))
+			m.logger.Info("New block.", "size", bytesProducedN)
```

Comment:
I would remove the size value
change it to more useful values (height, num of tx)

Reply:
i removed this log and added size to the existing block created log in produceblock func